src/vm/arm64/stubs.cpp

```diff
@@ -866,43 +843,48 @@ void HijackFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
 {
     LIMITED_METHOD_CONTRACT;
 
-    _ASSERTE(!"ARM64:NYI");
+     pRD->IsCallerContextValid = FALSE;
+     pRD->IsCallerSPValid      = FALSE;
+ 
+     pRD->pCurrentContext->Pc = m_ReturnAddress;
+     size_t s = sizeof(struct HijackArgs);
+     _ASSERTE(s%8 == 0); // HijackArgs contains register values and hence will be a multiple of 8
+     // stack must be multiple of 16. So if s is not multiple of 16 then there must be padding of 8 bytes 
+     s = s + (s%16 != 0) ? 8 : 0; 
```

Comment:
We already know that s is a multiple of 8 here, so you can write:

s = s + (s%16)